frontend/src/bundles/common/components/button/button.tsx

```diff
@@ -1,15 +1,68 @@
-type Properties = {
+import { type ReactNode } from 'react';
+
+import { getValidClassNames } from '../../helpers/get-valid-class-names.js';
```

Comment:
```suggestion
import { getValidClassNames } from '../../helpers/helpers.ts';
```
Move this helper to a separate directory and re-export it from `helpers.ts`

Reply:
Additionally, please, update the typings for the helper function. You can either use `typeof clsx` or `ClassValue` type provided by the package.